AWS CLI - Idaptive V1/AWSCLI.py

```diff
@@ -31,7 +31,9 @@ def get_environment(args):
         tenant = tenant + ".centrify.com"
```

Comment:
Just curious, why still keeping `centrify.com` here?

Reply:
1. Remove "and "centrify.com" not in tenant"
2. Use ".idaptive.app" instead ".centrify.com"
3. Make sure, story requires "Doc changes". https://identity-developer.cyberark.com/docs/making-cacertspem step is not needed if "cert = True" used (no certificate pinning) which is default now.